src/fragmentarium/application/FragmentService.ts

```diff
@@ -36,6 +36,7 @@ export interface FragmentRepository {
     references: ReadonlyArray<Reference>
   ): Promise<Fragment>
   folioPager(folio: Folio, fragmentNumber: string): Promise<any>
+  fragmentPager(fragmentNumber: string): Promise<any>
```

Comment:
There should be a type for the pager data. We should not add more `any`s. You should configure your IDE to show *eslint* warnings, and fix them whenever possible.

Reply:
Is there need to add type in withData function ? In FragmentPager.tsx for example ?